Multi Nonce begone! (#28)
ietf-rats/draft-birkholz-rats-epoch-marker: draft-birkholz-rats-epoch-markers.md

This is the sha-256 hash of the string "EPOCH_BELL".

### Epoch Tick {#sec-epoch-tick}

> An Epoch Tick is a single opaque blob send to multiple consumers.

Reviewer: ```suggestion
An Epoch Tick is a single opaque blob sent to multiple consumers.
```

~~~~ CDDL
{::include cddl/multi-nonce.cddl}
~~~~

The following describes the epoch-tick type.

epoch-tick:

> : Either a string, a byte string, or an integer used by RATS roles in a trust domain as extra data included in the production of conceptual messages as specified by the RATS architecture {{-rats-arch}} to associate them with a certain epoch.

Reviewer: ```suggestion
: Either a string, a byte string, or an integer used by RATS roles within a trust domain as extra data included in conceptual messages {{-rats-arch}} to associate them with a certain epoch.
```
